src/pkg/cli/cert.go

```diff
@@ -102,7 +102,7 @@ func GenerateLetsEncryptCert(ctx context.Context, client cliClient.Client) error
 		}
 	}
 	if cnt == 0 {
-		term.Infof("No HTTPS services found; no TLS cert generation needed")
+		term.Infof("No HTTPS domains found in compose file; no TLS cert generation needed")
```

Comment:
Good call, I think we can make it more user friendly by showing the actual field name:
```suggestion
		term.Infof("No `domainname` found in compose file; no HTTPS cert generation needed")
```
Also, TLS is not user facing, so no need to mention it.